src/tools/kata-ctl/src/utils.rs

```diff
@@ -144,6 +144,12 @@ pub fn get_generic_cpu_details(cpu_info_file: &str) -> Result<(String, String)>
     Ok((vendor, model))
 }
 
+const VHOST_VSOCK_DEVICE: &str = "/dev/vhost-vsock";
```

Comment:
looks like `VHOST_VSOCK_DEVICE ` is never used, if that's correct, could you remove it please?